### PR TITLE
Documentation: Fixed broken links

### PR DESCRIPTION
**Pull request by kickoke**
##### Summary

The deploy log in Netlify returned the following broken links: 

```
6:34:19 PM: - On source page path = /docs/agent/claim:
6:34:19 PM:    -> linking to /docs/agent/packaging/distributions
6:34:19 PM: 
6:34:19 PM: - On source page path = /docs/agent/packaging/installer/methods/kickstart:
6:34:19 PM:    -> linking to /docs/agent/packaging/installer/methods/packaging/installer/methods/macos
6:34:19 PM: 
6:34:19 PM: - On source page path = /docs/agent/packaging/installer/methods/macos:
6:34:19 PM:    -> linking to /docs/agent/packaging/installer/methods/claim#troubleshooting (resolved as: /docs/agent/packaging/installer/methods/claim)
6:34:19 PM: 
6:34:19 PM: - On source page path = /guides/step-by-step/step-99:
6:34:19 PM:    -> linking to /docs/agent/#contribute (resolved as: /docs/agent/)
```

This PR tries to fix the broken links.

### CODE REVIEW

File: docs/guides/step-by-step/step-99.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ If that feels like too much possibility to you, why not one of these options:
 -   Share your experience with Netdata and this guide. Be sure to [@mention](https://twitter.com/linuxnetdata) us on 
     Twitter!
 -   Contribute to what we do. Browse our [open issues](https://github.com/netdata/netdata/issues) and check out out
-    [contributions doc](/README.md#contribute) for ideas of how you can pitch in.
+    [contributions doc](/contribute) for ideas of how you can pitch in.
```
Comment (ilyam8) [inline docs/guides/step-by-step/step-99.md:47]:
```suggestion
    [contributions doc](https://learn.netdata.cloud/contribute/) for ideas of how you can pitch in.
```

`/contribute` doesn't work on GitHub, how about that^^?

Comment (kickoke) [reply docs/guides/step-by-step/step-99.md:47]: @ilyam8 Well, that is a good point. 
Do other relative file links work as long as they are in the same repo?

Comment (ilyam8) [reply docs/guides/step-by-step/step-99.md:47]: > relative
> as they are in the same repo

Yes.

File: claim/README.md [hunk 2]
```diff
@@ -460,8 +460,8 @@ using the [ACLK](/aclk/README.md).
 
 The best way to install Netdata and connect your nodes to Netdata Cloud is with our automatic one-line installation script, [kickstart](/packaging/installer/README.md#automatic-one-line-installation-script). This script will install the Netdata Agent, in case it isn't already installed, and connect your node to Netdata Cloud. 
 
-This  works with:
-* all Linux distributions, see [Netdata distribution support matrix](https://learn.netdata.cloud/docs/agent/packaging/distributions)
+This works with:
+* most Linux distributions, see [Netdata's platform support policy](/docs/agent/packaging/platform_support)
```
Comment (ilyam8) [inline claim/README.md:464]:
```suggestion
* most Linux distributions, see [Netdata's platform support policy](/packaging/PLATFORM_SUPPORT.md)
```

Comment (ilyam8) [reply claim/README.md:464]: > /docs/agent/packaging/

I think that is the path in netdata/learn repo.

**Review by ilyam8 — APPROVED** [review]

**Review by DShreve2 — APPROVED** [review]

**Pull request merged**